### ISSUE TITLE
Advertise WebCal calendar URLs (feature request)

### ISSUE THREAD

**Issue by belidzs** [Kozea/Radicale#776]
According to [this](https://www.davdroid.com/manual/accounts/#c205) a CalDav server can advertise Webcal calendars. These are typically used to access read-only, shared calendars.

> DAVdroid recognizes Webcal calendars in the calendar home set which are published with resourcetype: subscribed and shows them in the DAVdroid account activity. If you select such a Webcal collection for synchronization, DAVdroid passes the URL to an installed Webcal-capable app like ICSdroid so that this app can subscribe to the calendar.

It would be nice if I could add these calendar URLs to my account so DAVdroid and other clients would display those extra calendars automagically.

**Comment by ialokim** [comment]
Would be really great to see this functionality added!

Some research (e.g. [here]()) showed me the "advertisement" should at least contain the following xml:
```xml
 <d:response>
  <d:href>/<user>/<calendar-name>/</d:href>
  <d:propstat>
   <d:prop>
    <d:resourcetype>
     <d:collection/>
     <cs:subscribed/>
    </d:resourcetype>
    <cs:source>
     <d:href>http://www.somesite.com/public-calendar.ics</d:href>
    </cs:source>
   </d:prop>
   <d:status>HTTP/1.1 200 OK</d:status>
  </d:propstat>
 </d:response>
```

So, instead of `<c:calendar/>`, it should contain `<cs:subscribed/>` and an `<cs:source>` tag for specifying the (public) ics file. (Where 'cs' would be the calendarserver namespace, e.g. `xmlns:cs="http://calendarserver.org/ns/"`.)

**Comment by leso-kn** [comment]
The subscribed calendars could be stored inside a single `.props` file in the root of each collection (e.g. `.subscriptions.props`).

Like this each user could have their own subscriptions in a multi user instance - given that each collection refers to one user.

**Comment by leso-kn** [comment]
Any update on this? @oliviakozea

**Comment by leso-kn** [comment]
Any update on this? @Kozea

**Comment by dorchain** [comment]
I second the request.

**Comment by MrPixelized** [comment]
I, too, second the request.

**Comment by maartenweyns** [comment]
Came here to leave a comment, this feature would be highly appreciated!

**Comment by tunbridgep** [comment]
I would also really like this feature.

**Comment by MrPixelized** [comment]
Wow! I'm really shocked to see that the changes are that simple. Great work! :D

**Comment by tunbridgep** [comment]
From my understanding you can subscribe to web calendars (like a public holidays one made by someone else and posted online) using a simple url. Having that functionality in the UI would definitely be nice.

**Comment by mmBesar** [comment]
This is the only feature missing, for me, in this great project.

**Comment by JadeVane** [comment]
I second the request. There are still many people waiting for this feature after such a long time, I believe this feature needs to be put on the agenda.

**Comment by 6801318d8d** [comment]
@Kozea can we at least merge the PR [leso-kn](https://github.com/leso-kn) #1229?

I second this FR

**Comment by pbiering** [comment]
https://github.com/Kozea/Radicale/pull/1229 merged into 3.2-devel
